neuro_logging/trace.py

```diff
@@ -34,9 +36,6 @@ async def new_sentry_trace_cm(
             except asyncio.CancelledError:
                 transaction.set_status("cancelled")
                 raise
-            except Exception as exc:
```

Comment:
let's keep it for new_trace, this is a top level span, it should capture exception